packages/mds-transactions-service/repository/index.ts

```diff
@@ -57,11 +78,22 @@ class TransactionReadWriteRepository extends ReadWriteRepository {
     }
   }
 
-  // TODO search criteria, paging
-  public getTransactions = async (search: TransactionSearchParams): Promise<TransactionDomainModel[]> => {
+  // TODO search criteria
+  public getTransactions = async (
+    search: TransactionSearchParams
+  ): Promise<{ transactions: TransactionDomainModel[]; cursor: Cursor }> => {
     const { connect } = this
-    const { provider_id, start_timestamp, end_timestamp } = search
-    function when(): { timestamp?: FindOperator<number> } {
+    const {
+      provider_id,
+      start_timestamp,
+      end_timestamp,
+      before,
+      after,
+      limit,
+      order
+    } = validateTransactionSearchParams(search)
+
+    const resolveTimeBounds = (): { timestamp?: FindOperator<number> } => {
```

Comment:
I must confess I'm not entirely sure what this is doing.

Reply:
It's just turning the `start_timestamp` and `end_timestamp` values into a clause that TypeORM can use for temporal bounds, so if there are both you want your timestamps to be sammiched between them, if you only have a start then you want it to be `>`, and if you only have an end then you want it to be `<`. Is that any more clear?

Reply:
👍